### DISCUSSION TITLE
Is Android Auto locking out OsmAnd ?

### DISCUSSION THREAD

**Discussion by mdykierek** [osmandapp/OsmAnd#15967] [Q&A]
Haven't driven using Android Auto for a while now. With OsmAnd 4.3.2 I decided to give it a try to see how it works and I am disappointed with what happened.
When I plug the phone and try to use OsmAnd on car's display, it reads "OsmAnd needs Location permission to navigate". When the engine is on I cannot use the button Grant access since it responds with "Not available while driving".
So take this scenario. I grant, start the engine (then Android Auto gets disconnected) and guess what - I am no longer able to use OsmAnd...
![1671019591486](https://user-images.githubusercontent.com/27214784/207591499-c667d92f-6511-4cbc-a90c-d6ddbd11da7e.jpg)
Any remedy? Anyone?
Sygic doesn't have this problem. Google Maps and Fuelio eiter.

Android Auto version 8.5.624594-release

**Comment by vshcherb** [comment]
https://github.com/osmandapp/OsmAnd/issues/15965
